### PR TITLE
NX-10-Created navigation

### PR DESCRIPTION
**Pull request by yulia-maleeva**
Created public and private routes. When it is private route and user is not logged it redirects to login page.

I changed some routes according to the API. I drew pictures: they are in comments on the project-board.

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,7 +1,33 @@
...
+
+import Login from "./components/pages/Login";
+import Home from "./components/pages/Home";
+import Movies from "./components/pages/Movies";
+import Trending from "./components/pages/Trending";
+import Movie from "./components/pages/Movie";
+
+import PrivateRoutes from "./components/pages/PrivateRoutes";
+import UserProfile from "./components/pages/UserProfile";
+import Favourites from "./components/pages/Favourites";
```
Comment (marinabudz) [inline src/App.tsx:14]: rename components to ui

File: src/App.tsx [hunk 2]
```diff
@@ -1,7 +1,33 @@
...
+
+import NotFound from "./components/pages/NotFound";
+
+const App: FC = () => (
+  <Routes>
+    <Route path={ROUTES.LOGIN} element={<Login />} />
```
Comment (marinabudz) [inline src/App.tsx:20]: must be availbale only for user who is not logged in 
Rename Login to Sign In
add Sign Up page route

File: src/components/pages/PrivateRoutes/index.tsx [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import React, { FC } from "react";
+
+import { Outlet, Navigate } from "react-router-dom";
+import ROUTES from "../../../constants/routes";
+
+const PrivateRoutes: FC = () => {
+  let isAuthorized: boolean = false;
```
Comment (marinabudz) [inline src/components/pages/PrivateRoutes/index.tsx:7]: rename to const

File: src/constants/routes/index.tsx [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+  LOGIN: string;
+  HOME: string;
+  MOVIES: string;
+  TRENDING: string;
+  MOVIE: string;
+  USER_PROFILE: string;
+  FAVOURITES: string;
+}
+
+const ROUTES: IRoutes = {
```
Comment (marinabudz) [inline src/constants/routes/index.tsx:11]: change to enum

File: src/constants/routes/index.tsx [hunk 5]
```diff
@@ -0,0 +1,21 @@
+interface IRoutes {
```
Comment (marinabudz) [inline src/constants/routes/index.tsx:1]: remove interface

**Review by marinabudz — CHANGES_REQUESTED** [review]

File: src/App.tsx [hunk 6]
```diff
@@ -1,7 +1,37 @@
...
+
+import { PrivateRoutes, PrivateRoute } from "./UI/pages/PrivateRoutes";
+import UserProfile from "./UI/pages/UserProfile";
+import Favourites from "./UI/pages/Favourites";
+
+import NotFound from "./UI/pages/NotFound";
+
+const App: FC = () => (
+  <Routes>
+    <Route path={ROUTES.SIGN_UP} element={<SignUp />} />
```
Comment (marinabudz) [inline src/App.tsx:21]: sign up should be also priovate

File: src/UI/pages/PrivateRoutes/index.tsx [hunk 7]
```diff
@@ -0,0 +1,14 @@
+import React, { FC } from "react";
+
+import { Outlet, Navigate } from "react-router-dom";
+import ROUTES from "../../../constants/routes";
+
+const isAuthorized: boolean = false;
+
+export const PrivateRoutes: FC = () => {
```
Comment (marinabudz) [inline src/UI/pages/PrivateRoutes/index.tsx:8]: rename as it is not obsious what it should do 
better AuthRoutes and NonAuthRoutes

**Review by marinabudz — CHANGES_REQUESTED** [review]

File: src/UI/pages/PrivateRoutes/index.tsx (hunk 7 above)
Comment (stellar-maryna) [reply src/UI/pages/PrivateRoutes/index.tsx:8]: AuthRoute and NonAuthRoute

**Review by marinabudz — APPROVED** [review]

**Pull request merged**